app/services/idv/steps/ipp/ssn_step.rb

```diff
@@ -18,6 +23,10 @@ def extra_view_variables
 
         private
 
+        def ssn
+          flow_params[:ssn]
+        end
```

Comment:
This is one of those little cleanups that makes a way bigger impact than one might expect. 👍